### DISCUSSION TITLE
SEO

### DISCUSSION THREAD

**Discussion by joms** [fremtind/jokul#2021] [Annet]
Når man googler "Designsystem", er Jøkul helt nede på 5. plass for meg. Hva kan vi gjøre for å dra Jøkul opp på topp? Her mistenker jeg vi må studere litt SEO og hvordan gjøre portalen vår kulere for Google.

Lighthouse gir oss 91 av 100 i kategorien SEO. Så jeg mistenker at om vi utnytter meta taggene kan vi hente en del. Her tror jeg kanskje vi kan gjøre noen tricks for deling av sider i sosiale medier også.

![image](https://user-images.githubusercontent.com/537060/121421541-8fbc4d00-c96e-11eb-8e48-cd1f73da0831.png)

Jeg vil tro vi har noen SEO-eksperter på huset. Om noen vet om noen er det bare å peke de hit 🤩

**Comment by lars-tafjord** [comment]
Hei, lighthouse gir bare en score basert på generell teknisk SEO. Når Google indekserer sider på søkeord (som "designsystem" i dette tilfelle) så spør crawler-roboten deres over 200 spørsmål. Blant annet spør den om søkeordet kommer frem i: 

- H1-overskrift
- Page-title
- Brødtekst
- Meta-beskrvelsen
- Alt-tekst i bilder
- H2 og H3 underoverskrifter. 
- Ankertekst i linker
- Synonymer
- +++

I tillegg har man backlinks. Har siden blitt linket til fra eksterne domener med høy autoritet? Hvor mange har delt siden i sosiale medier?Er siden godt linket internt? Hvordan er den teknisk tilstand på siden? Hva er alderen på domenet osv. 

Denne gir et godt overblikk over teknisk tilstand på en enkeltside, hvor også Core Web Vitals kommer frem (som er ekstra viktig i disse dager):
https://developers.google.com/speed/pagespeed/insights

MEN, før man starter og optimalisere en side for et søkeord, bør man se på intensjonen bak søket designsystem. Er intensjonen å finne frem til Fremtinds designsystem, er det å finne et designsystem for inspirasjon, en mal for å bygge sitt eget designsystem? 

Et kjapt søk i keywordtool.io på "designsystem" gir meg over 240 søkeord knyttet til designsystem. Lager man en søkeordsanalyse med en intensjonsvurdering av de søkeordene ser man hvilke relaterte søkeord som eksisterer, hvor mange ganger de søkes etter hver måned og hvilke søkeord man bør fokusere på. Det gir et tydeligere svar på intensjonen og kanskje er det ikke "designsystem" som er det viktigste søkeordet for denne siden likevel? 

Man bør også vite hva man ønsker å få igjen fra trafikken man henter inn til siden. Når vi optimaliserer en produktside (f.eks. bilforsikring på sparebank1.no) så ønsker vi at de besøkende skal kjøpe bilforsikring, finne svar på spørsmål de ellers ville kontaktet rådgiver for å få, samt hjelp til å melde skade osv. Hva ønsker man fra de som besøker Jøkul-siden? :)
